t/mojo/morbo.t

```diff
@@ -90,6 +91,12 @@ EOF
   ok $tx->is_finished, 'transaction is finished';
   is $tx->res->code, 200,            'right status';
   is $tx->res->body, 'Hello World!', 'right content';
+
+  # This new one is there
```

Comment:
I don't understand this comment. Same below.

Reply:
Ooops.  It was my fiddling with code I am discovering! Will remove.